testsuite/features/support/cobbler_test.rb

```diff
@@ -12,18 +12,28 @@
 
 # Class for clobber test
 class CobblerTest
+  ##
+  # It creates a new XMLRPC client object, and then checks to see if the server is running
   def initialize
     server_address = ENV['SERVER']
     @server = XMLRPC::Client.new2('http://' + server_address + '/cobbler_api', nil, DEFAULT_TIMEOUT)
     raise 'No running server at found at ' + server_address unless running?
   end
 
+  ##
+  # It logs into cobbler and returns the token
```

Comment:
```suggestion
  # It logs into Cobbler and returns the token
```
Cobbler is always spellt with a capital C.